shell/src/main/java/alluxio/cli/fsadmin/report/CapacityCommand.java

```diff
@@ -189,6 +174,84 @@ private void printAggregatedInfo(GetWorkerReportOptions options) {
     }
   }
 
+  /**
+   * Prints worker capacity information.
+   *
+   * @param workerInfoList the worker info list to get info from
+   */
+  private void printWorkerInfo(List<WorkerInfo> workerInfoList) {
+    mIndentationLevel = 0;
+    if (mCapacityTierInfoMap.size() == 0) {
+      return;
+    } else if (mCapacityTierInfoMap.size() == 1) {
+      // Do not print Total value when only one tier exists
+      printShortWorkerInfo(workerInfoList);
+      return;
+    }
+
+    String workerInfoFormat = "%-16s %-16s %-13s %-16s %s";
+    StringBuilder tiersInfo = new StringBuilder();
+    Set<String> tiers = mCapacityTierInfoMap.keySet();
+    tiersInfo.append(String.format(Strings.repeat(" %-13s", tiers.size()), tiers.toArray()));
+    print(String.format("%n" + workerInfoFormat,
+        "Worker Name", "Last Heartbeat", "Storage", "Total", tiersInfo.toString()));
+
+    for (WorkerInfo info : workerInfoList) {
+      String workerName = info.getAddress().getHost();
+
+      long usedBytes = info.getUsedBytes();
+      long capacityBytes = info.getCapacityBytes();
+
+      String usedPercentageInfo = "";
+      if (capacityBytes != 0) {
+        int usedPercentage = (int) (100L * usedBytes / capacityBytes);
+        usedPercentageInfo = String.format(" (%s%%)", usedPercentage);
+      }
+
+      StringBuilder capacityTierInfo = new StringBuilder();
+      StringBuilder usedTierInfo = new StringBuilder();
+
+      for (Map.Entry<String, Map<String, String>> entry : mCapacityTierInfoMap.entrySet()) {
+        capacityTierInfo.append(String.format(" %-13s",
```

Comment:
this adds an unwanted space at the start. For combining a bunch of strings using a separator java has https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html.

Reply:
Changed to
```java
  /**
   * Gets the formatted tier values of a worker.
   *
   * @param map the map to get worker tier values from
   * @param workerName name of the worker
   * @return the formatted tier values of the input worker name
   */
  private static String getWorkerFormattedTierValues(Map<String, Map<String, String>> map,
      String workerName) {
    return map.entrySet().stream().map((entry)
        -> (String.format("%-14s", entry.getValue().getOrDefault(workerName, "-"))))
        .collect(Collectors.joining(""));
  }
```
Thanks for the good advice!